packages/docusaurus/lib/default-theme/Loading/styles.module.css

```diff
@@ -1,9 +1,9 @@
-/**
- * Copyright (c) 2017-present, Facebook, Inc.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
+/**	
```

Comment:
Seems like random spaces added here. Did you configure your editor to strip trailing spaces?